src/components/Terminal/TerminalTabs.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+
+import { TerminalInfo } from './TerminalInfo';
+import state from '../../state';
+
+export interface TerminalTab {
+  name: string;
+  tab: React.ReactNode;
+  content: React.ReactNode;
+}
+
+interface TerminalTabsProps {
+  tabs: Array<TerminalTab>;
+  active?: string;
+}
+
+export const TerminalTabs: React.FunctionComponent<TerminalTabsProps> = ({
+  tabs = [],
+  active = 0,
+}) => {
+  const [activeTab, setActiveTab] = useState(active);
+  const editorState = state.useEditorState();
+
+  if (tabs.length === 0) {
+    return null;
+  }
+
+  return (
+    <div>
+      <div
+        className="flex flex-row justify-between items-center px-2 border-b border-gray-700 text-white uppercase font-bold text-xs cursor-pointer"
+        onClick={e => {
+          const clientRects = e.currentTarget.parentElement?.parentElement?.getClientRects()[0];
```

Comment:
🧐

Reply:
Yes 😅  I need to get how many pixels away from the bottom of an element so that the terminal closes itself on click - I am taking a few pixel adjustments to improve the UX.